generator/source/faq/validator.rst

```diff
@@ -248,8 +248,12 @@ What does this error mean: ``sawtooth-validator[... ERROR cli] Cannot have a gen
 ------------------------------------------------------------------------------------------------------------------------
 You tried to create a new genesis block when you did not need to (because there already is a genesis block). To solve, this remove file ``/var/lib/sawtooth/genesis.batch.file`` and restart ``sawtooth-validator`` .
 
-I get this error when testing with a lot of validators: ``Max occupancy was not provided by transaction processor: ... Using default max occupancy: 10``
+I get this warning when running a transaction processor: ``Max occupancy was not provided by transaction processor: ... Using default max occupancy: 10``
 --------------------------------------------------------------------------------------------------------------------------------------------------------
+There is no need to worry about this warning message, at least not the smart contract developers. Max occupancy is the number of transactions that a transaction processor can handle simultaneously. Note that the validator would send transactions to the transaction processor only when all the dependencies are resolved. By default the value is set to 10. If there are more than 10 transactions available, then the validator would wait for the transaction processor's occupancy be available to schedule the execution. This is introduced so that the transaction processor is not overwhelmed with the requests.
```

Comment:
Changes:
- "at least not the smart..." => "at least not as a smart..."
- "This is introduced so that the transaction processor is not overwhelmed..." => "This feature prevents the validator being overwhelmed..."

Reply:
Done